.circleci/images/primary/Dockerfile-2.6.7

```diff
@@ -1,5 +1,6 @@
-# https://hub.docker.com/layers/ruby/library/ruby/2.6.7-buster/images/sha256-88239eaf531f72ca1f641379393228acd399d7db01bee5d566b79276406b4fc6
-FROM ruby@sha256:88239eaf531f72ca1f641379393228acd399d7db01bee5d566b79276406b4fc6
+# This image supports multiple platforms, see README for build instructions
+
+FROM ruby:2.6.7
```

Comment:
Minor: Consider moving this to Ruby 2.6.10 :)

Reply:
I propose we do each Ruby version updates in a dedicated PR, first to minimise impact, and also they would require updating the appraisal files.

Reply:
Done in https://github.com/DataDog/dd-trace-rb/pull/2402